pdp/__init__.py

```diff
@@ -56,14 +57,26 @@ def get_config():
             'js/pdp_vector_map.js'
             ], debug=(not config['js_min'])),
         'templates': resource_filename('pdp', 'templates'),
-        'version': get_distribution('pdp').version
+        'version': parse_version("version"),
+        'revision': parse_version("revision")
         }
 
     config = updateConfig(global_config, config)
     if config['session_dir'] == 'default':
         config['session_dir'] = resource_filename('pdp', 'pdp_session_dir')
     return config
 
+def parse_version(type):
+    full_version = get_distribution('pdp').version
```

Comment:
If you split out the get_distribution call to be outside of this function, it makes it easier to test (b/c you can pass in an arbitrary version as a parameter in the tests).